samza-sql/src/test/java/org/apache/samza/sql/avro/schemas/ComplexUnion.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Autogenerated by Avro
+ * 
+ * DO NOT EDIT DIRECTLY
+ */
+package org.apache.samza.sql.avro.schemas;  
```

Comment:
I am wondering if this can be auto generated by the Build/Testing framework at runtime ? using something like `avro-maven-plugin`  here is an example https://github.com/alexholmes/avro-maven.
FYI this is a suggestion comment and can be tackled as a followup if it is has a big scope.

Reply:
I agree that auto-generating the avro-code using gradle plugin at build-time would be better and i'll create a follow-up ticket for it.